ui/src/app/components/help-sidebar.tsx

```diff
@@ -164,14 +164,15 @@ const iconStyles = {
   }
 };
 // TODO uncomment 'thunderstorm' icon when cluster configuration function is ready
-const icons = [{
-  id: 'help',
-  disabled: false,
-  faIcon: faInfoCircle,
-  label: 'Help Icon',
-  page: null,
-  style: {fontSize: '21px'},
-  tooltip: 'Help Tips',
+const icons = (helpContent) => {
```

Comment:
I'd add a type annotation for the function param at least. Even better, a one-line comment describing the 'helpContent' parameter and why it's needed.

[optional] If you're game for as mass-rename, I think s/helpContent/helpContentKey/ in this file would help reduce confusion throughout, since AFAICT this is really a key into the foreign JSON file which contains the actual content.